Latest update breaks the plugin

**Plugin version:**
2.0.32

Hi @smockle - I know you're no longer supporting this plugin, but I thought I'd send this to you just in case you can help.  I updated the plugin from 2.0.30 to 2.0.32, and now it crashes homebridge.  Error attached.  Any thoughts?  Thanks in advance for your help!

![image](https://user-images.githubusercontent.com/46855061/69201163-5d889780-0b0b-11ea-894c-92dd751c71a8.png)

I think I am seeing this also. Attached is a Balena OS log showing this (or a similar) failure:

```
01.12.19 15:37:22 (-0500) Restarting service 'main sha256:7ab4690946fbba70672aa5b1f22541178d183b6cdc27f40cff8bfe5bbc088c12'
01.12.19 15:37:19 (-0500) <main> 2019-12-01T20:37:25.272195000Z 
01.12.19 15:37:25 (-0500) <main> > balena-homebridge@0.1.0 start /usr/src/app
01.12.19 15:37:25 (-0500) <main> > npm run start:setup && npm run start:homebridge
01.12.19 15:37:25 (-0500) <main> 
01.12.19 15:37:27 (-0500) <main> 
01.12.19 15:37:27 (-0500) <main> > balena-homebridge@0.1.0 start:setup /usr/src/app
01.12.19 15:37:27 (-0500) <main> > ./setup.sh
01.12.19 15:37:27 (-0500) <main> 
01.12.19 15:37:27 (-0500) <main> Homebridge is already configured at "/data/.homebridge/".
01.12.19 15:37:27 (-0500) <main> Run "Purge Data" and then "Restart" action to reset to default configuration.
01.12.19 15:37:30 (-0500) <main> 
01.12.19 15:37:30 (-0500) <main> > balena-homebridge@0.1.0 start:homebridge /usr/src/app
01.12.19 15:37:30 (-0500) <main> > DEBUG=* homebridge --debug --user-storage-path /data/.homebridge --plugin-path /usr/src/app/node_modules/
01.12.19 15:37:30 (-0500) <main> 
01.12.19 15:37:34 (-0500) <main> [12/1/2019, 8:37:34 PM] Loaded config.json with 0 accessories and 2 platforms.
01.12.19 15:37:34 (-0500) <main> [12/1/2019, 8:37:34 PM] ---
01.12.19 15:37:34 (-0500) <main> [12/1/2019, 8:37:34 PM] Loaded plugin: homebridge-config-ui-x
01.12.19 15:37:34 (-0500) <main> [12/1/2019, 8:37:34 PM] Registering platform 'homebridge-config-ui-x.config'
01.12.19 15:37:34 (-0500) <main> [12/1/2019, 8:37:34 PM] ---
01.12.19 15:37:34 (-0500) <main> [12/1/2019, 8:37:34 PM] ====================
01.12.19 15:37:34 (-0500) <main> [12/1/2019, 8:37:34 PM] ERROR LOADING PLUGIN homebridge-lutron-caseta-smockle:
01.12.19 15:37:34 (-0500) <main> [12/1/2019, 8:37:34 PM] Error: Cannot find module './lutron-caseta-platform'
01.12.19 15:37:34 (-0500) <main> Require stack:
01.12.19 15:37:34 (-0500) <main> - /usr/src/app/node_modules/homebridge-lutron-caseta-smockle/src/index.js
01.12.19 15:37:34 (-0500) <main> - /usr/src/app/node_modules/homebridge/lib/plugin.js
01.12.19 15:37:34 (-0500) <main> - /usr/src/app/node_modules/homebridge/lib/server.js
01.12.19 15:37:34 (-0500) <main> - /usr/src/app/node_modules/homebridge/lib/cli.js
01.12.19 15:37:34 (-0500) <main> - /usr/src/app/node_modules/homebridge/bin/homebridge
01.12.19 15:37:34 (-0500) <main>     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:800:15)
01.12.19 15:37:34 (-0500) <main>     at Function.Module._load (internal/modules/cjs/loader.js:693:27)
01.12.19 15:37:34 (-0500) <main>     at Module.require (internal/modules/cjs/loader.js:864:19)
01.12.19 15:37:34 (-0500) <main>     at require (internal/modules/cjs/helpers.js:74:18)
01.12.19 15:37:34 (-0500) <main>     at Object.<anonymous> (/usr/src/app/node_modules/homebridge-lutron-caseta-smockle/src/index.js:2:34)
01.12.19 15:37:34 (-0500) <main>     at Module._compile (internal/modules/cjs/loader.js:971:30)
01.12.19 15:37:34 (-0500) <main>     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1011:10)
01.12.19 15:37:34 (-0500) <main>     at Module.load (internal/modules/cjs/loader.js:822:32)
01.12.19 15:37:34 (-0500) <main>     at Function.Module._load (internal/modules/cjs/loader.js:730:14)
01.12.19 15:37:34 (-0500) <main>     at Module.require (internal/modules/cjs/loader.js:864:19)
01.12.19 15:37:34 (-0500) <main> [12/1/2019, 8:37:34 PM] ====================
01.12.19 15:37:34 (-0500) <main> [12/1/2019, 8:37:34 PM] Loading 2 platforms...
01.12.19 15:37:34 (-0500) <main> [12/1/2019, 8:37:34 PM] [Homebridge Config UI] Initializing config platform...
01.12.19 15:37:34 (-0500) <main> [12/1/2019, 8:37:34 PM] [Homebridge Config UI] Spawning homebridge-config-ui-x with PID 75
01.12.19 15:37:34 (-0500) <main> [12/1/2019, 8:37:34 PM] Error: The requested platform 'LutronCasetaPlatform' was not registered by any plugin.
01.12.19 15:37:34 (-0500) <main>     at API.platform (/usr/src/app/node_modules/homebridge/lib/api.js:134:13)
01.12.19 15:37:34 (-0500) <main>     at Server._loadPlatforms (/usr/src/app/node_modules/homebridge/lib/server.js:327:45)
01.12.19 15:37:34 (-0500) <main>     at Server.run (/usr/src/app/node_modules/homebridge/lib/server.js:90:36)
01.12.19 15:37:34 (-0500) <main>     at module.exports (/usr/src/app/node_modules/homebridge/lib/cli.js:59:10)
01.12.19 15:37:34 (-0500) <main>     at Object.<anonymous> (/usr/src/app/node_modules/homebridge/bin/homebridge:17:22)
01.12.19 15:37:34 (-0500) <main>     at Module._compile (internal/modules/cjs/loader.js:971:30)
01.12.19 15:37:34 (-0500) <main>     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1011:10)
01.12.19 15:37:34 (-0500) <main>     at Module.load (internal/modules/cjs/loader.js:822:32)
01.12.19 15:37:34 (-0500) <main>     at Function.Module._load (internal/modules/cjs/loader.js:730:14)
01.12.19 15:37:34 (-0500) <main>     at Function.Module.runMain (internal/modules/cjs/loader.js:1051:12)
01.12.19 15:37:34 (-0500) <main> [12/1/2019, 8:37:34 PM] Got SIGTERM, shutting down Homebridge...
01.12.19 15:37:40 (-0500) <main> npm ERR! code ELIFECYCLE
01.12.19 15:37:40 (-0500) <main> npm ERR! errno 143
01.12.19 15:37:40 (-0500) <main> npm ERR! balena-homebridge@0.1.0 start:homebridge: `DEBUG=* homebridge --debug --user-storage-path /data/.homebridge --plugin-path /usr/src/app/node_modules/`
01.12.19 15:37:40 (-0500) <main> npm ERR! Exit status 143
01.12.19 15:37:40 (-0500) <main> npm ERR! 
01.12.19 15:37:40 (-0500) <main> npm ERR! Failed at the balena-homebridge@0.1.0 start:homebridge script.
01.12.19 15:37:40 (-0500) <main> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
01.12.19 15:37:40 (-0500) <main> 
01.12.19 15:37:40 (-0500) <main> npm ERR! A complete log of this run can be found in:
01.12.19 15:37:40 (-0500) <main> npm ERR!     /root/.npm/_logs/2019-12-01T20_37_40_091Z-debug.log
01.12.19 15:37:40 (-0500) <main> npm ERR! code ELIFECYCLE
01.12.19 15:37:40 (-0500) <main> npm ERR! errno 143
01.12.19 15:37:40 (-0500) <main> npm ERR! balena-homebridge@0.1.0 start: `npm run start:setup && npm run start:homebridge`
01.12.19 15:37:40 (-0500) <main> npm ERR! Exit status 143
01.12.19 15:37:40 (-0500) <main> npm ERR! 
01.12.19 15:37:40 (-0500) <main> npm ERR! Failed at the balena-homebridge@0.1.0 start script.
01.12.19 15:37:40 (-0500) <main> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
01.12.19 15:37:40 (-0500) <main> 
01.12.19 15:37:40 (-0500) <main> npm ERR! A complete log of this run can be found in:
01.12.19 15:37:40 (-0500) <main> npm ERR!     /root/.npm/_logs/2019-12-01T20_37_40_212Z-debug.log
01.12.19 15:37:41 (-0500) <main> 
01.12.19 15:37:43 (-0500) Service exited 'main sha256:7ab4690946fbba70672aa5b1f22541178d183b6cdc27f40cff8bfe5bbc088c12'
```

--- a/.npmignore
+++ b/.npmignore
@@ -2,4 +2,3 @@ __tests__
 .github
 coverage
 node_modules
-src

